src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
+  image="canonical-usage.png",
+  caption="Canonical adoption has increased to 60.6% on mobile.",
+  description="A column chart highlighting percentages of pages which have a canonical set or are canonicalized.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=34038632&format=interactive",
+  sheets_gid="1931314929",
+  sql_file="pages-canonical-stats.sql"
+  )
+}}
+
+We see from the data that canonical tags usage has increased over the years. In 2019, 48.3% of mobile pages used canonicals. In 2020, this grew to 53.6%. In 2021, this grew even further to 58.5%. And in 2022, it’s increased to 60.6%.
+
+Mobile has a higher percentage of canonical attribution than desktop (60.6% vs. 58.7%), which is likely a direct result of single use URLs on mobile. Since the data set in this chapter is limited to home pages, it’s fair to assume that this is the reason for the higher canonical attribution on mobile. According to  [Google’s guidelines](https://developers.google.com/search/mobile-sites/mobile-seo/separate-urls), having a separate mobile site is not recommended.
+
+### HTML vs. HTTP canonical usage
+
+There are two ways of implementing canonical tags:
+
+1. Within the HTML `<head>`
+2. Within the HTTP headers (`Link` HTTP header)
+
+{{ figure_markup(
+  image="html-versus-http-canonical.png",
+  caption="HTML canonicals are the primary implementation with 60.4% appearing in the page markup.",
+  description="A column chart comparing canonical implementations between HTML and HTTP headers.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=1403695913&format=interactive",
+  sheets_gid="1931314929",
+  sql_file="pages-canonical-stats.sql"
+  )
+}}
+
+The most common usage across both desktop and mobile is through HTML at 58.6% and 60.4%, respectively. This is likely due to the ease of implementation. While one requires basic HTML knowledge, the other method (through HTTP headers) requires a more technical skillset.
+
+### Raw vs. rendered usage
+
+{{ figure_markup(
+  image="raw-vs-rendered-canonical.png",
+  caption="1% of canonicals are rendered but were not received in the raw HTML.",
+  description="A bar chart comparing raw, rendered, or http canonical statuses.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=473608668&format=interactive",
+  sheets_gid="1931314929",
+  sql_file="pages-canonical-stats.sql"
+  )
+}}
+
+Compared to 2021, where raw canonical usage was 57.7% and rendered canonical usage was 58.4%, in 2022 there was some growth, with raw canonical usage reaching 59.4% and rendered canonical usage rising to 60.4%. This correlates with the growth in overall canonical use.
+
+## Page experience
+
+### HTTPS
```

Comment:
We don't like hanging headers like this. Can we add a bit of text between them?

Reply:
Can we add: 

In this section of the chapter, we're looking at different elements of page experience and how this has evolved since the 2021 Web Almanac.

Reply:
Added